### PR TITLE
set allow_overlapping_ips in all.yml

### PR DESCRIPTION
**Pull request by ssrikant20**

### CODE REVIEW

**Comment by craigtracey** [comment]
:+1:

**Comment by j2sol** [comment]
ok to test

**Comment by craigtracey** [comment]
ok to test

**Comment by j2sol** [comment]
Good to merge once tests pass, although I suspect this one is going to need a chunk of documentation to go with it.

File: roles/neutron-common/templates/etc/neutron/neutron.conf [hunk 1]
```diff
@@ -21,7 +21,7 @@ service_plugins = {{ neutron.service_plugins|union(neutron.lbaas.service_plugin)
 {% elif neutron.service_plugins|length %}
 service_plugins = {{ neutron.service_plugins|join(',') }}
 {% endif %}
-allow_overlapping_ips = False
+allow_overlapping_ips = {{ neutron.allow_overlapping_ips }}
```
Comment (dlundquist) [inline roles/neutron-common/templates/etc/neutron/neutron.conf]: `| bool` might be a good idea.

**Pull request merged**